src/app/states/db/page.mdx

```diff
@@ -0,0 +1,51 @@
+export const metadata = {
+  title: 'db',
+  description: 'An app state of database synchronization. in Inkdrop. This section explains how to fetch and manipulate the `notes` state.',
```

Comment:
This state has nothing to do with notes. This is probably an error in the old documentation.

```suggestion
  description: 'An app state of database synchronization. in Inkdrop. This section explains how to fetch and manipulate the `db` state.',
```